common/djangoapps/student/views/dashboard.py

```diff
@@ -836,7 +836,7 @@ def student_dashboard(request):
         'consent_required_courses': consent_required_courses,
         'enterprise_customer_name': enterprise_customer_name,
         'enrollment_message': enrollment_message,
-        'redirect_message': redirect_message,
+        'redirect_message': Text(redirect_message),
```

Comment:
Do any of the other messages need the same treatment?

Reply:
don't know, if you're referring to messages on the dashboard not familiar with all the vulnerable codepaths used there